lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -0,0 +1,136 @@
+define((require, exports, module) ->
+  'use strict'
+  oop = require('../../lib/oop')
+  TextHighlightRules = require('../../mode/text_highlight_rules').TextHighlightRules
+
+  PapeeriaLatexHighlightRules = ->
+    ###*
+    * correct environment into stack
```

Comment:
what's the meaning of stak in this context? Is it a stack of nested semantic constructions, like document=>list=>list=>maths ?